src/store/sagas/index.ts

```diff
@@ -7,6 +7,21 @@ import { positionsSaga } from '@sagas/positions'
 import { statsHandler } from './stats'
 import { farmsSaga } from './farms'
 import { bondsSaga } from './bonds'
+import { RpcStatus, actions as solanaConnectionActions } from '@reducers/solanaConnection'
+import { put, select } from 'typed-redux-saga'
+import { rpcStatus } from '@selectors/solanaConnection'
+
+export function* handleError(error: Error): Generator {
```

Comment:
I think we could change the name of this to e.g. `handleRpcError` ro sth similar